docs/install.md

```diff
@@ -13,29 +13,29 @@ The use of a virtual environment for scikit-decide is recommended, and you will
 This can be achieved either by using [conda](https://docs.conda.io/en/latest/) or by using [pyenv](https://github.com/pyenv/pyenv) (or [pyenv-win](https://github.com/pyenv-win/pyenv-win) on windows)
 and [venv](https://docs.python.org/fr/3/library/venv.html) module.
 
-The following examples show how to create a virtual environment with Python version 3.8.11 with the mentioned methods.
+The following examples show how to create a virtual environment with Python version 3.8.13 with the mentioned methods.
 
 #### With conda (all platforms)
 
 ```shell
-conda create -n skdecide python=3.8.11
+conda create -n skdecide python=3.8.13
```

Comment:
Is there a reason for that ?

Reply:
Yes, (as i said in the commit message ;p) the install was not working with conda (probably because of dependencies conflicts i do not now). But it worked with python 3.8.13 so i chose to change everywhere the python version in the example so that one can copy/paste and have it work.

Reply:
my bad, didn't open the commit description. then lgtm